NDTensors/src/linearalgebra.jl

```diff
@@ -402,11 +402,98 @@ function LinearAlgebra.qr(T::DenseTensor{ElT,2,IndsT}; kwargs...) where {ElT,Ind
   q = dim(q) < dim(r) ? sim(q) : sim(r)
   Qinds = IndsT((ind(T, 1), q))
   Rinds = IndsT((q, ind(T, 2)))
-  Q = tensor(Dense(vec(Matrix(QM))), Qinds)
+  Q = tensor(Dense(vec(Matrix(QM))), Qinds) #Q was strided
   R = tensor(Dense(vec(RM)), Rinds)
   return Q, R
 end
 
+#
+#  Uses kwargs:positive to decide which rq method to call.
+#
+function rq(T::DenseTensor{ElT,2,IndsT}; kwargs...) where {ElT,IndsT}
+  if get(kwargs, :positive, false)
+    RM, QM = rq_positive(matrix(T))
+  else
+    RM, QM = rq(matrix(T))
+  end
```

Comment:
Move `positive` into the function signature.

Reply:
What about:
```julia
function rq(T::DenseTensor{ElT,2,IndsT}; positive=false, kwargs...) where {ElT,IndsT}
  # [...]
end
```

Reply:
Done